var/spack/repos/builtin/packages/py-python-clamd/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPythonClamd(PythonPackage):
```

Comment:
I would rename this package `py-clamd` since that's what name they use on PyPI. That's why I always download my Python packages from PyPI.